src/Components/ChildComponent.js

```diff
@@ -0,0 +1,33 @@
+// import React from 'react';
+
+function Child() {
+  return (
+    <div className="keypad">
+      <button type="button" className="btn">AC</button>
+      <button type="button" className="btn">+/-</button>
+      <button type="button" className="btn">%</button>
+      <button type="button" className="operatorBtn">÷</button>
+
+      <button type="button" className="btn">7</button>
+      <button type="button" className="btn">8</button>
+      <button type="button" className="btn">9</button>
+      <button type="button" className="operatorBtn">x</button>
+
+      <button type="button" className="btn">4</button>
+      <button type="button" className="btn">5</button>
+      <button type="button" className="btn">6</button>
+      <button type="button" className="operatorBtn">-</button>
+
+      <button type="button" className="btn">1</button>
+      <button type="button" className="btn">2</button>
+      <button type="button" className="btn">3</button>
+      <button type="button" className="operatorBtn">+</button>
+
+      <button type="button" className="btn0">0</button>
+      <button type="button" className="btn">.</button>
+      <button type="button" className="operatorBtn">=</button>
+    </div>
```

Comment:
- [OPTIONAL] Kindly make sure to name your component a well-detailed name. You can name `Buttons` instead of `Child`. Because in some scenarios you may have many child components so it's better to provide a well-detailed name.